tm2838/Atelier
# Release 02.22.2022

## Description

Nice day for a release!
- Synchronize main with dev branch since a lot of refactoring and cleanup happened

## How to test

- Start up the application locally
- Start up product overview service and product review service and their corresponding databases locally
- Verify that everything is loaded on the page

## Notes


## Issue tracking
